router/route/user.go

```diff
@@ -121,15 +122,104 @@ func retrieveUser(w http.ResponseWriter, r *http.Request) (int, error) {
 func retrieveUserScore(w http.ResponseWriter, r *http.Request) (code int, err error) {
 	username := mux.Vars(r)["username"]
 	score, err := model.GetUserScore(username)
```

Comment:
The `err` here is ignored and overwritten with `nil` by the next function call below.

Reply:
Is this check removed by a bad merge?

Reply:
Hmm, yes this might be an artifact of the merge. Your code was inserted right into a function of mine. Tried to do it right.